modules/Makefile.circleci

```diff
@@ -16,13 +16,10 @@ circle\:deps:
 
 ## Tag using branch + build num and push to registry (CircleCI)
 circle\:tag:
-	@$(call assert_set,CIRCLE_BRANCH)
-	@$(call assert_set,CIRCLE_BUILD_NUM)
+	@$(call assert_set,BUILD)
```

Comment:
Ah yes, more pain to change then its worth.

Update the target description to reflect the new behavior, then :+1: